KsApi/models/graphql/adapters/Project+ProjectFragment.swift

```diff
@@ -48,6 +59,27 @@ extension Project {
   }
 }
 
+private func projectPersonalization(isStarred: Bool,
+                                    backing: Backing?,
+                                    friends: [User],
+                                    projectBackingUserId: String?) -> Project.Personalization {
+  var currentUserIsBacker: Bool {
+    guard let userId = projectBackingUserId,
+      let userIdValue = Int(userId) else {
+      return false
+    }
+
+    return backing?.backerId == userIdValue
```

Comment:
Don't think we need this ID comparison, if the `Project` has a `Backing` then the current user is backing it.